docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -222,62 +224,113 @@ endif::[]
 You can import an event schema or API definition into {product-long-api-designer} from a URL, from a file, or from an {product-long-registry} instance.
 
 .Prerequisites
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
-* You can access a running {product-registry} instance in your organization, if you want to import from {registry}.
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
+* You can access a running {product-registry} instance in your organization, if you want to import from {product-long-registry}.
+For more information, see {base-url}{getting-started-url-registry}[Getting started with {product-long-registry}^].
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click *Import a schema or API design*, and then click one of the following options:
+. In the {service-url-api-designer}[{product-long-api-designer} web console^], click *Import a schema or API design*, and then click one of the following options:
 .. *Import from file*: Click *Browse* and select a file, or drag and drop a file.
```

Comment:
Think these options should be an unordered (that is bullet), list.

Reply:
Done